k9mail-library/src/test/java/com/fsck/k9/mail/K9LibRobolectricTestRunner.java

```diff
@@ -0,0 +1,20 @@
+package com.fsck.k9.mail;
+
+import org.junit.runners.model.InitializationError;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+public class K9LibRobolectricTestRunner extends RobolectricTestRunner {
+
+    public K9LibRobolectricTestRunner(Class<?> testClass) throws InitializationError {
+        super(testClass);
+    }
+
+    @Override
+    protected Config buildGlobalConfig() {
+        return new Config.Builder()
+                .setSdk(21)
+                .setManifest(Config.NONE)
```

Comment:
I think some tests might need a config - it's how you get resources. Again I'm too novice to know whether there's a downside to always providing one.